packages/fast-usdc/src/utils/address.js

```diff
@@ -0,0 +1,57 @@
+import { makeError, q } from '@endo/errors';
+
+/**
+ * Very minimal 'URL query string'-like parser that handles:
```

Comment:
do we not have access to `URL` in XS?

Reply:
Glad to see the tests.

[qs-lite.min.js](https://github.com/kawanet/qs-lite/blob/master/dist/qs-lite.min.js) in  https://github.com/kawanet/qs-lite is 530 bytes. Maybe easier to source that to get a more conventional qs ability

Reply:
Cool, xs seems to have `decodeURIComponent` too: https://github.com/Moddable-OpenSource/moddable/blob/978945595c6aaa460c6068b34d3ebc415db248a7/xs/sources/xsCommon.c#L1375

Reply:
I suppose we can get by without `decodeURIComponent` for now but consider mentioning in the code that it's available in XS